### DISCUSSION TITLE
increasing the number of pinned repos?

### DISCUSSION THREAD

**Discussion by niXman** [community/community#42374] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hello!

my question is: is there any way how can I increase the number of pinned repos?

best!

**Comment by SilasRodrigues19** [comment]
Hi @niXman 

There is a discussion on this topic at the link below where an alternative was suggested that allows pinning more than 6 repositories:

 [https://github.com/orgs/community/discussions/28350](https://github.com/orgs/community/discussions/28350)

**Comment by niXman** [comment]
ah, got it! thank you!

but nevertheless I think it would be great if that functionality be builtin in github...

**Reply by SilasRodrigues19** [reply]
Really, but in that case I think you have to ask in the "Product Feedback" category.

**Reply by hzshmar** [reply]
@

**Reply by hzshmar** [reply]
H

**Reply by hzshmar** [reply]
hzshmar668@gmail.com
